### ISSUE TITLE
Upgrade ROS on TurtleBot to the latest version

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#36]
## User story
1. As a SD
2. I want to updtae from ROS 1 to ROS 2 
3. to be able to use packages (such as kubuki_node for reading out the battery level) without the need to backport and to fullfil the given project constraints by the customer.

## Acceptance criteria
* Latest version of ROS is on TurtleBot

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Sebastian2023** [comment]
https://docs.ros.org/en/foxy/Installation.html
+
https://github.com/SICKAG/sick_lidar_localization
https://github.com/kobuki-base/kobuki_ros

**Comment by Sebastian2023** [comment]
Still WIP.

**Comment by Sebastian2023** [comment]
I upgraded both TurtleBots to ROS 2 Galactic Geochelone. sick_lidar_localization worked out of the box but kobuki_ros needed some workarounds. Demonstrated in #6. 

https://docs.ros.org/en/humble/Releases/Release-Galactic-Geochelone.html
https://github.com/kobuki-base/kobuki_ros/issues/41
https://github.com/SICKAG/sick_lidar_localization
